fw/Core/Hitcon/App/SpaceshipApp.cc

```diff
@@ -33,17 +31,13 @@ void SpaceshipApp::OnEntry() {
 }
 // 2. OnExit
 void SpaceshipApp::OnExit() {
-  if (_routine_task.IsEnabled()) {
-    scheduler.DisablePeriodic(&_routine_task);
-  }
+  scheduler.DisablePeriodic(&_routine_task);
```

Comment:
Why are you removing the check here? You're certain it won't happen? Note that disabling an already disabled periodic task will assert.

Reply:
Let me restore it.